### PR TITLE
fix docs in gru.py

### PR DESCRIPTION
**Pull request by hlthu**

### CODE REVIEW

File: egs/wsj/s5/steps/libs/nnet3/xconfig/gru.py [hunk 1]
```diff
@@ -962,8 +963,8 @@ def generate_pgru_config(self):
         # formulation for OPGRU like:
         # z_t = \sigmoid ( x_t * U^z + s_{t-1} * W^z ) // update gate
         # o_t = \sigmoid ( x_t * U^o + s_{t-1} * W^o ) // output gate
-        # \tilde{h}_t = \tanh ( x_t * U^h + y_{t-1} \dot W^h ) // W^h is learnable vector
-        # h_t = ( 1 - z_t ) \dot h_t + z_t \dot y_{t-1}
+        # h_t = \tanh ( x_t * U^h + y_{t-1} \dot W^h ) // W^h is learnable vector
```
Comment (GaofengCheng) [inline egs/wsj/s5/steps/libs/nnet3/xconfig/gru.py]: why '\tilde' is deleted?

Comment (hlthu) [reply egs/wsj/s5/steps/libs/nnet3/xconfig/gru.py]: you have `\tilde{h}_t` on the left, but never used. ie, you defined it by:
```
# \tilde{h}_t = \tanh ( x_t * U^h + y_{t-1} \dot W^h ) // W^h is 
```
but in the next:
```
# h_t = ( 1 - z_t ) \dot h_t + z_t \dot y_{t-1}
```
` \dot h_t ` is a mistake, it must be  `\dot \tilde{h}_t `; also, `dot y_{t-1}` should be `dot h_{t-1}`, so I just use `h_t` to replace  `\tilde{h}_t` and `y_t` to replace `h_t`.
It is ok to use the old one, but need to change the following line:
```
# h_t = ( 1 - z_t ) \dot h_t + z_t \dot y_{t-1}
```
into
```
# h_t = ( 1 - z_t ) \dot \tilde{h}_t + z_t \dot h_{t-1}
```

Comment (hlthu) [reply egs/wsj/s5/steps/libs/nnet3/xconfig/gru.py]: Aslo, these notations are the same as them in the codes.

Comment (GaofengCheng) [reply egs/wsj/s5/steps/libs/nnet3/xconfig/gru.py]: Oh, OK, change all the formulation as the one in SPL paper (GRU, PGRU, OPGRU), you should have one. Let's uniform them right now.

Comment (GaofengCheng) [reply egs/wsj/s5/steps/libs/nnet3/xconfig/gru.py]: The code could be slightly different from formulation.

**Pull request merged**